modules/programs/micro.nix

```
@@ -9,12 +9,19 @@ let
   jsonFormat = pkgs.formats.json { };
 
 in {
-  meta.maintainers = [ hm.maintainers.mforster ];
+  meta.maintainers = [ hm.maintainers.mforster maintainers.pbsds ];
 
   options = {
     programs.micro = {
       enable = mkEnableOption "micro, a terminal-based text editor";
 
+      trueColor = mkOption {
```

Perhaps

```suggestion
      enableTrueColor = mkOption {
```

since we often use the `enable` prefix for Boolean options.